static/app/views/insights/database/views/databaseSpanSummaryPage.spec.tsx

```diff
@@ -37,6 +40,10 @@ describe('DatabaseSpanSummaryPage', function () {
     },
   });
 
+  jest.mocked(useParams).mockReturnValue({
+    groupId,
+  });
```

Comment:
So I had to mock this because our new widget grabs groupId from `useParams` -- however the summary page is passed params via props via router